src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -803,6 +803,84 @@ public function testMappingDuplicateKeyFlow()
         $this->assertSame($expected, Yaml::parse($input));
     }
 
+    /**
+     * @dataProvider getParseExceptionOnDuplicateData
```

Comment:
Not sure we need it, the test will fail anyway when the deprecation will be removed.